src/schema/objects/metadata.yaml

```diff
@@ -2698,6 +2772,7 @@ _StandardTemplateDeprecatedCoordSys:
   - UNCInfant2V23
 _iEEGCoordSys:
   type: string
+  # TODO: Add definitions for these values.
```

Comment:
What do the different `_iEEGCoordSys` values mean?

Reply:
see https://github.com/bids-standard/bids-specification/pull/919/files#r785334647